index.html

```diff
@@ -43,5 +43,162 @@ <h2 class="heading-2">I am Software Developer</h2>
             </ul>
         </div>
     </section> 
+
+        <!-- works -->
+
+    <section class="Works">
+        <div class="works-title" >
+            <h3 class="heading-3">My Recent Works </h3> 
+            <hr class="works-line">
+        </div>
+        <div class="works-img">
+            <img src="./images/works/img-works-0.svg" alt="works">
+        </div>
+        <div class="stories">
+            <h1 class="stories-headings">Multi-Post Stories</h1>
+            <p class="stories-content">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+    
+        <div class="Stories-lang">
+            <ul class="stories-lang-list">
+                <li class="stories-list-1">css</li>
+                <li class="stories-list-2">html</li>
+                <li class="stories-list-3">bootstrap</li>
+                <li class="stories-list-4">Ruby</li>
+            </ul>
+        </div>
+        <button type="submit" class="button-list">See Project</button>
+            
+        <div class="articles">
+            <div class="article-1">
+                <h4 class="heading-article"> Profesional Art Printing Data</h4>
+                  <p class="heading-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                
+                  <div class="Stories-lang-article-1">
+                      <ul class="stories-lang-list">
+                          <li class="stories-list-article">html</li>
+                          <li class="stories-list-article">bootstrap</li>
+                          <li class="stories-list-article">Ruby</li>
+                      </ul>
+                  </div>
+            </div>
+        <button type="submit" class="button-articles">See Project</button>
```

Comment:
- [ ] My comment on the type of the button, applies to all of the buttons in this section.